### DISCUSSION TITLE
Heart of Asia

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#655] [Q&A]
**Channel Name:** Heart of Asia
**Channel ID:** HeartofAsiaChannel.ph
**Country:** Philippines
**Language:** Tagalog
**External Link (optional):** xxx
**Notes (optional):** Free to air channel in the Philippines.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19656
